question1.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<head>
```

Comment:
Properly indent your html files, similar to code, if a html tag is inside another one